src/main/java/net/neoforged/neoforge/client/model/NeoForgeModelProperties.java

```diff
@@ -26,4 +27,9 @@ private NeoForgeModelProperties() {}
      * Render type to use. For block models, this can be specified under the {@code render_type} JSON key.
      */
     public static final ContextKey<RenderTypeGroup> RENDER_TYPE = ContextKey.vanilla("render_type");
+
+    /**
+     * Part visibilities. For models with named parts (i.e. OBJ and composite), this can be specified under the {@code visibility} JSON key
```

Comment:
Do we want this to be inherited? If yes this is fine. If not it doesn't need to be here.

Reply:
I think inheriting it makes sense because it allows stuff like disabling all parts in the base model and then selectively enabling the necessary parts in the child models.